src/client/clang_command.cc

```diff
@@ -100,6 +100,11 @@ bool ClangCommand::FillFlags(base::proto::Flags* flags,
              arg->getOption().matches(OPT_D) ||
              arg->getOption().matches(OPT_I)) {
       arg->render(arg_list_, non_cached_list);
+    } else if (arg->getOption().matches(OPT_include_pch) ||
+               arg->getOption().matches(OPT_include_pth)) {
+      auto* included_file = flags->add_included_files();
+      *included_file = arg->getValue();
+      arg->render(arg_list_, non_cached_list);
```

Comment:
Even without manual rendering still I don't see the point to render it inside `non_cached_list` - it's already counted inside `manifest.headers` for direct cache, and `non_cached_list` is not applied for simple cache.